pkg/transform/transformconfig/transform_config.go

```diff
@@ -66,5 +66,9 @@ func GetTransformations(c *config.Config, supportedTransformations []config.Conf
 		funcs = append(funcs, f)
 	}
 
+	if c.Data.Metrics.E2ELatencyEnabled {
```

Comment:
I actually think it needs to first in the list, rather than last - unless we make some other changes (which we might do in future):

a) transformations mutate the data, 

combined with 

b) we don't always use a snowplow transformation that provides the intermediate state (rare, but it happens!)

means we can have a situation whereby at the end, we can't grab a collector tstamp. But as long as the data is Snowplow enriched events, we can always do it at the beginning (and this transformation will never affect a downstream one).

Reply:
Oh yeah, you're right! Good spot